src/main/sensors/sensors.h

```diff
@@ -49,6 +49,7 @@ typedef enum {
     SENSOR_SONAR = 1 << 4,
     SENSOR_GPS = 1 << 5,
     SENSOR_GPSMAG = 1 << 6,
+    SENSOR_PITOT = 1 << 7,
```

Comment:
Shouldn't we also report PITOT via MSP to be able to show the cool-looking icon in the Configurator?